public/js/SearchIcon.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable react/jsx-no-undef */
+/* eslint-disable no-undef */
+
+class SearchIcon extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      searchName: ''
+    };
+
+    this.handleInputChange = this.handleInputChange.bind(this);
+  }
+
+  handleInputChange(event) {
+    const target = event.target;
+    const value =
+      target.type === 'checkbox'
+        ? target.checked
+        : target.type === 'number'
+          ? +target.value
+          : target.value;
+
+    let { searchName } = this.state;
+
+    searchName = value.toUpperCase();
+
+    this.setState({
+      searchName
+    });
+    this.props.searchSymbolWithName(searchName);
+  }
+
+  render() {
+    document.addEventListener("touchstart", function () { }, true);
```

Comment:
What does this line do?

Reply:
I used that code from the search box that I found. I didn't try without it though. I think it sets the event listener to the search box? But from what I know it should work without it. Let me test :)

Reply:
Yeah, I don't think that line is necessary, but yeah, test it out. :)